python-frontend/src/main/java/org/sonar/python/semantic/SymbolTableBuilder.java

```diff
@@ -232,18 +236,18 @@ public void visitImportFrom(ImportFrom pyImportFromTree) {
       String moduleName = moduleTree != null
         ? moduleTree.names().stream().map(Name::name).collect(Collectors.joining("."))
         : null;
-      createImportedNames(pyImportFromTree.importedNames(), moduleName, !pyImportFromTree.dottedPrefixForModule().isEmpty());
+      createImportedNames(pyImportFromTree.importedNames(), moduleName, pyImportFromTree.dottedPrefixForModule());
```

Comment:
The semantic of `dottedPrefixForModule` is confusing and it's not documented (but that's not a new issue).